### DISCUSSION TITLE
Where is the WebRoot for AdGuard?

### DISCUSSION THREAD

**Discussion by vlsalsa** [AdguardTeam/AdGuardHome#5354] [Q&A]
I have installed AdGuard properly on my home network on a Raspberry Pi 3. 

I am working on trying to install an SSL certificate to access it over a domain and a cloudflare tunnel. As I go to install a certificate with Certbot, I am being asked on the Raspberry pi for the webroot location...

where does the webroot install on a standard installation of AdGuard on the Raspberry Pi?

**Comment by Disqu3-mirOir-qt** [comment]
use standalone instead webroot

sudo certbot certonly --standalone
